### PR TITLE
Fix unit tests and re-enable tests in CI

### PR DESCRIPTION
**Pull request by levinmr**

### CODE REVIEW

**Comment by levinmr** [comment]
There are still a few unit tests to fix here.  Not ready for merge yet.

File: test/process-results/ga-data-processor.test.js [hunk 1]
```diff
@@ -103,6 +171,7 @@ describe("GoogleAnalyticsDataProcessor", () => {
       config.account.hostname = "www.example.gov"
 
       const result = GoogleAnalyticsDataProcessor.processData(report, dataWithHostname)
+      debugger;
```
Comment (scottqueen-bixal) [inline test/process-results/ga-data-processor.test.js:174]: do we want these debuggers?

File: test/process-results/result-formatter.test.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ describe("ResultFormatter", () => {
       return ResultFormatter.formatResult(result, { format: "csv", slim: true }).then(formattedResult => {
         const lines = formattedResult.split("\n");
         const [header, ...rows] = lines
+        debugger;
```
Comment (scottqueen-bixal) [inline test/process-results/result-formatter.test.js:47]: do we want these debuggers?

File: .mocharc.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+diff: true
+extension: ['js']
+package: './package.json'
+slow: '75'
+spec: 'test/**/*.js'
+timeout: '2000'
+ui: 'bdd'
+watch-files:
+  - 'src/**/*.js'
+  - 'test/**/*.js'
```
Comment (scottqueen-bixal) [inline .mocharc.yml:10]: Can we configure to include the test for the ua app?

```suggestion
  - 'test/**/*.js'
  - 'ua/src/**/*.js'
  - 'ua/test/**/*.js'
```

File: README.md [hunk 4]
```diff
@@ -365,6 +365,27 @@ Compose:
 docker-compose up
 ```
 
+#### Running unit tests locally
```
Comment (scottqueen-bixal) [inline README.md:368]: nice, worked perfectly

**Review by scottqueen-bixal — CHANGES_REQUESTED** [review]

**Review by scottqueen-bixal — APPROVED** [review]

**Pull request merged**